spec/support/i18n.rb

```diff
@@ -1,14 +1,3 @@
 RSpec.configure do |config|
   config.include AbstractController::Translation
```

Comment:
We could probably do away with this file by putting the `config.include AdstractController::Translation` into the `rails_helper.rb` as well.